### ISSUE TITLE
get_terminal_size was removed in click 8.1.0

### ISSUE THREAD

**Issue by petrchmelar** [cruft/cruft#151]
Since `get_terminal_size` function [was removed from click](https://click.palletsprojects.com/en/8.1.x/changes/#version-8-1-0) and cruft depends on `click = ">= 7.1.2, <9.0.0"`, the `cruft create` is failing with:

```
Traceback (most recent call last):
  File "/opt/venv/bin/cruft", line 5, in <module>
    from cruft._cli import app
  File "/opt/venv/lib/python3.8/site-packages/cruft/__init__.py", line 6, in <module>
    from cruft._commands import check, create, diff, link, update
  File "/opt/venv/lib/python3.8/site-packages/cruft/_commands/__init__.py", line 2, in <module>
    from .check import check
  File "/opt/venv/lib/python3.8/site-packages/cruft/_commands/check.py", line 5, in <module>
    import typer
  File "/opt/venv/lib/python3.8/site-packages/typer/__init__.py", line 12, in <module>
    from click.termui import get_terminal_size as get_terminal_size
ImportError: cannot import name 'get_terminal_size' from 'click.termui' (/opt/venv/lib/python3.8/site-packages/click/termui.py)
```

As suggested in click's changelog, the `get_terminal_size` call could be replaced by `shutil.get_terminal_size` maybe.

**Comment by petrchmelar** [comment]
Caused by https://github.com/tiangolo/typer/issues/377

**Comment by petrchmelar** [comment]
It will be probably fixed by merging https://github.com/tiangolo/typer/pull/375

**Comment by tiangolo** [comment]
Hello! I just released Typer `0.4.1` that should handle this. :rocket: :nerd_face:

**Comment by petrchmelar** [comment]
Thanks, cruft seems to be working now. :) closing
